.github/actions/javascript/contributorChecklist/contributorChecklist.js

```diff
@@ -0,0 +1,168 @@
+- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like \`Avatar\` is modified, I verified that \`Avatar\` is working as expected in all cases)
+- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
+- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.`;
+
+const issue = github.context.payload.issue ? github.context.payload.issue.number : github.context.payload.pull_request.number;
+const combinedData = [];
+
+function printUncheckedItems(result) {
+    const checklist = result.split('\n');
+
+    checklist.forEach((line) => {
+        // Provide a search string with the first 30 characters to figure out if the checkbox item is in the checklist
+        const lineSearchString = line.replace('- [ ] ', '').slice(0, 30);
+        if (line.includes('- [ ]') && (completedContributorChecklist.includes(lineSearchString) || completedContributorPlusChecklist.includes(lineSearchString))) {
+            console.log(`Unchecked checklist item: ${line}`);
+        }
+    });
+}
+
+// Get all user text from the pull request, review comments, and pull request comments
+GitHubUtils.octokit.pulls.get({
+    owner: GitHubUtils.GITHUB_OWNER,
+    repo: GitHubUtils.APP_REPO,
+    pull_number: issue,
+}).then(({data: pullRequestComment}) => {
+    combinedData.push(pullRequestComment.body);
+}).then(() => GitHubUtils.octokit.pulls.listReviews({
+    owner: GitHubUtils.GITHUB_OWNER,
+    repo: GitHubUtils.APP_REPO,
+    pull_number: issue,
+})).then(({data: pullRequestReviewComments}) => {
+    pullRequestReviewComments.forEach(pullRequestReviewComment => combinedData.push(pullRequestReviewComment.body));
+})
+    .then(() => GitHubUtils.octokit.issues.listComments({
+        owner: GitHubUtils.GITHUB_OWNER,
+        repo: GitHubUtils.APP_REPO,
+        issue_number: issue,
+        per_page: 100,
+    }))
+    .then(({data: pullRequestComments}) => {
+        pullRequestComments.forEach(pullRequestComment => combinedData.push(pullRequestComment.body));
+        let contributorChecklistComplete = false;
+        let contributorPlusChecklistComplete = false;
+
+        // Once we've gathered all the data, loop through each comment and look to see if it contains a completed checklist
+        for (let i = 0; i < combinedData.length; i++) {
+            const whitespace = /([\n\r])/gm;
+            const comment = combinedData[i].replace(whitespace, '');
+
+            if (comment.includes(completedContributorChecklist.replace(whitespace, ''))) {
+                contributorChecklistComplete = true;
+            } else if (comment.includes('- [')) {
+                printUncheckedItems(combinedData[i], completedContributorChecklist);
```

Comment:
Your function only takes one parameter right now, so this can just be `printUncheckedItems(combinedData[i]);` (or alternatively, could update the function to pass the checklist and only check against that passed list?). Either way.

Reply:
Yes, good catch!